### ISSUE TITLE
Error when using np.memmap

### ISSUE THREAD

**Issue by william-silversmith** [seung-lab/cloud-volume#441]
@ranlu reports that the following code errors:

```python
from cloudvolume import CloudVolume
import numpy as np
vol = CloudVolume("gs://ng_scratch_ranl/make_cv_happy/ws/134ac11bd90f993bb4f54bb7debd072f")
fp = np.memmap("chunk.data", dtype=np.uint64, mode='r', shape=(48,48,56), order='F')
vol[2000:2048, 2000:2048, 200:256]=fp[:]
```

```
python3.8/site-packages/cloudvolume/chunks.py in encode_compressed_segmentat
ion_c_ext(subvol, block_size)
    147     order = 'F'
    148     subvol = np.asfortranarray(subvol)
--> 149   return cseg.compress(subvol, block_size=block_size, order=order)
    150 
    151 def encode_compressed_segmentation_pure_python(subvol, block_size):
src/compressed_segmentation.pyx in compressed_segmentation.compress()
python3.8/site-packages/compressed_segmentation.cpython-38-x86_64-linux-gnu.
so in View.MemoryView.memoryview_cwrapper()
python3.8/site-packages/compressed_segmentation.cpython-38-x86_64-linux-gnu.
so in View.MemoryView.memoryview.__cinit__()
ValueError: buffer source array is read-only
```

**Comment by william-silversmith** [comment]
One way to fix this:

```python
  def do_upload(imgchunk, cloudpath):
    if type(imgchunk) == np.memmap:
      imgchunk = np.copy(imgchunk)
```

Not incredibly elegant.

**Comment by william-silversmith** [comment]
Error was introduced in https://github.com/seung-lab/cloud-volume/commit/89cce5dd2d18a927472c2698b2134faff8f662a6 when we stopped calling np.copy

**Comment by william-silversmith** [comment]
Resolved in #447
